Jenkinsfile

```diff
@@ -9,12 +9,22 @@ elifePipeline {
             }
 
             stage 'Build image', {
-                dockerBuild 'api-dummy', commit
+                sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml build"
             }
 
             stage 'Project tests', {
-                dockerBuildCi 'api-dummy', commit
-                dockerProjectTests 'api-dummy', commit
+                try {
+                    sh "docker run --name api-dummy_tests_${commit} elifesciences/api-dummy_ci:${commit}"
+                } finally {
+                    sh "docker cp api-dummy_tests_${commit}:/srv/api-dummy/build/. build"
+                    step([$class: "JUnitResultArchiver", testResults: 'build/phpunit.xml'])
+                    sh "docker rm api-dummy_tests_${commit}"
+                }
+                try {
+                    sh "IMAGE_TAG=${commit} docker-compose -f docker-compose.yml -f docker-compose.ci.yml up -d"
```

Comment:
Will return soon after the containers are started, so doesn't test much without a "wait and smoke test" phase afterwards

Reply:
The approach of https://github.com/elifesciences/annotations/blob/develop/Jenkinsfile#L27 usually works

Reply:
It'll have to be done from the outside as it replaces the `CMD` otherwise and fails.

Reply:
Can be added later, but usually `docker exec` runs in addition to the `CMD`?

Reply:
Yep, that turned out to be incorrect.